samples/getting-started/azure-quantum/Getting started with Qiskit and IonQ on Azure Quantum.ipynb

```diff
@@ -0,0 +1,652 @@
+      "text/plain": [
+       "<Figure size 504x360 with 1 Axes>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Submit the circuit to run on Azure Quantum\n",
+    "qpu_job = qpu_backend.run(circuit, shots=1024)\n",
+    "job_id = qpu_job.id()\n",
+    "print(\"Job id\", job_id)\n",
+    "\n",
+    "# Monitor job progress and wait until complete:\n",
+    "job_monitor(qpu_job)\n",
+    "\n",
+    "# Get the job results (this method also waits for the Job to complete):\n",
+    "result = qpu_job.result()\n",
+    "print(result)\n",
+    "counts = {format(n, \"03b\"): 0 for n in range(8)}\n",
+    "counts.update(result.get_counts(circuit))\n",
+    "print(counts)\n",
+    "plot_histogram(counts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82b0b1d8",
+   "metadata": {},
+   "source": [
+    "# Quantum Phase Estimation (QPE)\n",
+    "\n",
+    "The `azure-quantum[qiskit]` also supports more advanced examples. You can take any existing Qiskit code sample and by changing only a few lines of code, submit it to Azure Quantum.\n",
+    "\n",
+    "Let's use the QPE example from Qiskit textbook: https://qiskit.org/textbook/ch-algorithms/quantum-phase-estimation.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec9beeb2",
+   "metadata": {},
+   "source": [
+    "Below, all cells from the example notebook are copy-pasted. Feel free to give it a try yourself!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "b682e9b2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#initialization\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import math\n",
+    "\n",
+    "# importing Qiskit\n",
+    "from qiskit import IBMQ, Aer, transpile, assemble\n",
+    "from qiskit import QuantumCircuit, ClassicalRegister, QuantumRegister\n",
+    "\n",
+    "# import basic plot tools\n",
+    "from qiskit.visualization import plot_histogram"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "578fda5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def qft_dagger(qc, n):\n",
+    "    \"\"\"n-qubit QFTdagger the first n qubits in circ\"\"\"\n",
+    "    # Don't forget the Swaps!\n",
+    "    for qubit in range(n//2):\n",
```

Comment:
It's pretty confusing to refer to the index of a qubit as `qubit`; was that in the original code?

Reply:
I ended up taking this code out.